From 37748fb2175111e7ab04927520889430a0cf080f Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Sun, 27 Aug 2017 20:32:20 -0400
Subject: [PATCH 1002/1005] Use /lib/systemd/system for SYSTEMD_DIR by default

OpenMandriva's systemd units are installed in the non-UsrMerge variant
of the systemd paths. Thus, this needs to be adjusted to install units properly.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 23e5f889..9f7be6ee 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ if (NOT PYTHON_DESIRED)
 endif()
 
 SET( SYSCONFDIR /etc)
-SET( SYSTEMD_DIR /usr/lib/systemd/system)
+SET( SYSTEMD_DIR /lib/systemd/system)
 
 if (${PYTHON_DESIRED} STREQUAL "2")
 	FIND_PACKAGE (PythonInterp REQUIRED)
-- 
2.14.1

